adhocracy4/comments/serializers.py

```diff
@@ -10,16 +10,24 @@ class CommentSerializer(serializers.ModelSerializer):
     user_name = serializers.SerializerMethodField()
     user_profile_url = serializers.SerializerMethodField()
     is_deleted = serializers.SerializerMethodField()
-    ratings = serializers.SerializerMethodField()
+    ratings = serializers.ModelSerializer(many=True)
```

Comment:
Also not sure if changing the ratings serializer this way is ok, but necessary to make it readable as a nested field in the definition of fields below.

Reply:
We dont want to serialize the ratings as a nested fields, but only get the number of different ratings which is done in get_ratings() below. So please leave it as it and it should work fine?